.circleci/config.yml

```diff
@@ -7,27 +7,6 @@ executors:
     docker:
       - image: circleci/openjdk:8u222-stretch
 jobs:
-  upgrade:
```

Comment:
Are those steps no longer needed (upgrading, stocking and tagging the version) ? I don't see them anywhere.

Reply:
The only part where they're referenced is in a commented-out part, so nop, not necessary. 

(We commented the section because the upgrade part will need to set the user's name & email in git, commit and repush on master which can leads to a loop of builds)